build/Dockerfile.base

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:16.04
```

Comment:
Did this file get deleted and resurrected?  What did you guys do while I was out? 😒